scripts/build_docs_config.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+"""
+This script is used to populate the table of contents for the API in the mkdocs config file.
+"""
+
+import argparse
+from pathlib import Path
+from typing import Any, List
+
+from ruamel.yaml import YAML
+
+
+API_TOC_KEY = "API"
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("target_yaml")
+    parser.add_argument("source_yaml")
+    parser.add_argument("docs_root")
+    parser.add_argument("api_docs_path")
```

Comment:
Can you throw some meat at these bones, so that `build_docs_config.py -h` prints something useful?